src/shared/components/popover.jsx

```diff
@@ -40,21 +40,7 @@ class OutsideClickBoundary extends React.Component {
   }
 
   render() {
-    return (
-      <div
-        ref={(el) => { this.container = el }}
-        style={{
-          width: '100%',
-          height: '100%',
-          position: 'fixed',
-          zIndex: 1000,
```

Comment:
I'm with you that the position: fixed part is not necessary, but the z-index does need to be set to something like 1000. Otherwise we face this problem:

![screen shot 2019-01-07 at 9 39 37 am](https://user-images.githubusercontent.com/95735/50783598-36f99f00-1260-11e9-8c71-07ed9cd59ce2.png)

This can be fixed elsewhere, probably.

Reply:
Ah ok, maybe we do need the wrapper element after all then. I think we can use `position: absolute` here to not disturb the positioning. That will also create a new stacking order, dispensing with the need for zIndex:

https://developer.mozilla.org/en-US/docs/Web/CSS/position